Mathlib/Algebra/Order/Ring/Unbundled/Basic.lean

```diff
@@ -761,6 +761,19 @@ lemma two_mul_le_add_sq [ExistsAddOfLE α] [MulPosStrictMono α]
 
 alias two_mul_le_add_pow_two := two_mul_le_add_sq
 
+/-- Binary, squared, and division-free **arithmetic mean-geometric mean inequality**
+(aka AM-GM inequality) for linearly ordered commutative semirings. -/
+lemma four_mul_le_add_sq [ExistsAddOfLE α] [MulPosStrictMono α]
+    [ContravariantClass α α (· + ·) (· ≤ ·)] [CovariantClass α α (· + ·) (· ≤ ·)]
+    (a b : α) : 4 * a * b ≤ (a + b) ^ 2 := by
+  calc 4 * a * b
+    _ = 2 * a * b + 2 * a * b := by rw [mul_assoc, two_add_two_eq_four.symm, add_mul, mul_assoc]
+    _ ≤ a ^ 2 + b ^ 2 + 2 * a * b := by gcongr; exact two_mul_le_add_sq _ _
+    _ = a ^ 2 + 2 * a * b + b ^ 2 := by rw [add_right_comm]
+    _ = (a + b) ^ 2 := (add_sq a b).symm
+
+alias four_mul_le_add_pow_two := four_mul_le_add_sq
```

Comment:
```suggestion
@[inherit_doc] alias four_mul_le_add_pow_two := four_mul_le_add_sq
```
or something like that

Reply:
Seems to be causing the following issue:

```
error: ././././Mathlib/Algebra/Order/Ring/Unbundled/Basic.lean:775:2: invalid `[inherit_doc]` attribute, could not infer doc source
error: Lean exited with code 1
Some required builds logged failures:
- Mathlib.Algebra.Order.Ring.Unbundled.Basic
error: build failed
uncaught exception: object file '././.lake/build/lib/Mathlib.olean' of module Mathlib does not exist
```

Reply:
I couldn't find `@[inherit_doc]` in combinations with `alias` in mathlib (and the old code didn't have it for `two_mul_le_add_sq`, so I will just remove it again?